provider/datagen/src/bin/datagen.rs

```diff
@@ -221,6 +221,11 @@ fn main() -> eyre::Result<()> {
             .long("use-separate-crates")
             .help("--format=mod only: use types from individual `icu_*` crates instead of the `icu` meta-crate.")
         )
+        .arg(Arg::with_name("LIST_TO_UCPTRIE_BINARY")
```

Comment:
Practically this will be used as `--list-to-ucptrie-binary $(which ucptrie)`, so can we save the param and just search the `PATH`?

Reply:
I don't think people will have that binary installed? It's not a regular ICU4C binary, it's a custom ICU4X thing you have to build, and it's called `list_to_ucptrie`.

Reply:
We can document that it should be on the `PATH` if they don't want to use wasmer. I dislike adding more CLI arguments, and I dislike putting this on `SourceData`.

Reply:
OTOH i dislike implict stuff being picked up from PATH

(and also as far as Google3 is concerned I bet there will be pushback to using PATH there: build systems in general hate this kind of thing)